src/utils/datetime.cljs

```diff
@@ -209,6 +209,17 @@
       (previous-years? datetime)
       (.format ^js (date-fmt) datetime))))
 
+(defn full-date->short-date
```

Comment:
Please, cover utility functions with tests. I'd use the `are` macro to keep tests concise and assert lots of inputs.